src/Runners/DryRunner.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace AdamBrett\ShellWrapper\Runners;
+
+
+use AdamBrett\ShellWrapper\Command\CommandInterface;
+
+class DryRunner extends FakeRunner
+{
+    public function run(CommandInterface $command)
+    {
+        echo $command."\r\n";
```

Comment:
I didn't spot this first time around, could you replace this with the line ending constant `PHP_EOL` and put spaces around the dot?